src/test/java/bio/terra/integration/TestOperations.java

```diff
@@ -0,0 +1,141 @@
+        assertTrue("dataset create launch response is present", jobResponse.getResponseObject().isPresent());
+
+        DataRepoResponse<DatasetSummaryModel> datasetResponse =
+            dataRepoClient.waitForResponse(jobResponse, DatasetSummaryModel.class);
+        assertThat("dataset create is successful", datasetResponse.getStatusCode(), equalTo(HttpStatus.CREATED));
+        assertTrue("dataset create response is present", datasetResponse.getResponseObject().isPresent());
+        return datasetResponse.getResponseObject().get();
+    }
+
+    public void deleteTestDataset(String datasetId) throws Exception {
+        DataRepoResponse<JobModel> jobResponse =
+            dataRepoClient.delete("/api/repository/v1/datasets/" + datasetId, JobModel.class);
+
+        assertTrue("dataset delete launch succeeded", jobResponse.getStatusCode().is2xxSuccessful());
+        assertTrue("dataset delete launch response is present", jobResponse.getResponseObject().isPresent());
+
+        DataRepoResponse<DeleteResponseModel> deleteResponse =
+            dataRepoClient.waitForResponse(jobResponse, DeleteResponseModel.class);
+        assertGoodDeleteResponse(deleteResponse);
+    }
+
+    /**
+     * Ingests JSON data taking the defaults for the ingest specification
+     *
+     * @param studyId    - id of study to load
+     * @param tableName  - name of table to load data into
+     * @param datafile   - file path within the bucket from property integrationtest.ingestbucket
+     * @return ingest response
+     * @throws Exception
+     */
+    public IngestResponseModel ingestJsonData(String studyId, String tableName, String datafile) throws Exception {
+        String ingestBody = buildSimpleIngest(tableName, datafile);
+        DataRepoResponse<JobModel> postResponse = dataRepoClient.post(
+            "/api/repository/v1/studies/" + studyId + "/ingest",
+            ingestBody,
+            JobModel.class);
+
+        assertTrue("ingest launch succeeded", postResponse.getStatusCode().is2xxSuccessful());
+        assertTrue("ingest launch response is present", postResponse.getResponseObject().isPresent());
+
+        DataRepoResponse<IngestResponseModel> response =
+            dataRepoClient.waitForResponse(postResponse, IngestResponseModel.class);
+
+        assertThat("ingestOne is successful", response.getStatusCode(), equalTo(HttpStatus.OK));
+        assertTrue("ingestOne response is present", response.getResponseObject().isPresent());
+
+        IngestResponseModel ingestResponse = response.getResponseObject().get();
+        assertThat("no bad sample rows", ingestResponse.getBadRowCount(), equalTo(0L));
+        return ingestResponse;
+    }
+
+    private String buildSimpleIngest(String table, String filename) {
+        // TODO: REVIEWERS: I think append is better than escaping. Alternate is to include javax.json. Should we???
+        StringBuilder ingestBuilder = new StringBuilder()
```

Comment:
Could you instantiate an instance of IngestRequestModel with the correct values and then convert it to json?

Reply:
Yes, that sounds better.

Reply:
I put this and another code suggesting into a tracking ticket in Jira for someone's next pass through this code.